exercises/two-fer/two-fer.js

```diff
@@ -0,0 +1,7 @@
+var TwoFer = function () {};
```

Comment:
There is a discussion about including stubs or not for new users (#424). We haven't decided anything yet, but I'm ok with this stub, as this exercise should be one of the first exercises to be solved by users.